.github/README.md

```diff
@@ -182,10 +182,50 @@ In some circumstances, you may need to specify port numbers as part of the site
 * [Terminus](https://pantheon.io/docs/terminus) is used for CLI access to the Pantheon platform.
 * Enhanced security (separated web root and secure passwords with [wp-password-bcrypt](https://github.com/roots/wp-password-bcrypt))
 
+
 ## Environment variables and secrets
 
 This application uses a variety of ways to manage environment variables and secrets. All of these values are available within our shared LastPass account.
 
+**Please note!** There are two almost-identical Terminus plugins, which provide distinct workflows:
+
+- `terminus secret` (note the singular) is used for values that are used during build / deploy.
+- `terminus secrets` (note the plural) is used for values that are used by the WordPress application itself (after it has been deployed).
+
+
+### Build secrets
+
+Managed with: [**Terminus Secrets Manager** plugin](https://github.com/pantheon-systems/terminus-secrets-manager-plugin)
+
```

Comment:
Adding an install note similar to what we have for the other terminus plugin might be helpful (for context, I couldn't remember how it worked and flailed around a bit before just clicking into the GitHub and going "ah")

Reply:
I've added a similar sentence to what we've used for the other Terminus plugin - good suggestion. Going to merge now.